deployment/ccip/changeset/globals/config.go

```diff
@@ -34,29 +35,51 @@ const (
 )
 
 var (
+	// DefaultCommitOffChainCfg represents the default offchain configuration for the Commit plugin
+	// on _most_ chains. This should be used as a base for all chains, with overrides only where necessary.
+	// Notable overrides are for Ethereum, which has a slower block time.
 	DefaultCommitOffChainCfg = pluginconfig.CommitOffchainConfig{
-		RemoteGasPriceBatchWriteFrequency:  *config.MustNewDuration(30 * time.Minute),
-		TokenPriceBatchWriteFrequency:      *config.MustNewDuration(30 * time.Minute),
+		RemoteGasPriceBatchWriteFrequency:  *config.MustNewDuration(RemoteGasPriceBatchWriteFrequency),
+		TokenPriceBatchWriteFrequency:      *config.MustNewDuration(TokenPriceBatchWriteFrequency),
 		NewMsgScanBatchSize:                merklemulti.MaxNumberTreeLeaves,
-		MaxReportTransmissionCheckAttempts: 5,
+		MaxReportTransmissionCheckAttempts: 10,
 		RMNSignaturesTimeout:               6900 * time.Millisecond,
 		RMNEnabled:                         true,
 		MaxMerkleTreeSize:                  merklemulti.MaxNumberTreeLeaves,
 		SignObservationPrefix:              "chainlink ccip 1.6 rmn observation",
-		TransmissionDelayMultiplier:        1 * time.Minute,
+		// TransmissionDelayMultiplier for non-ETH (i.e, typically fast) chains should be pretty aggressive.
+		// e.g assuming a 2s blocktime, 15 seconds is ~8 blocks.
+		TransmissionDelayMultiplier:        15 * time.Second,
```

Comment:
This is currently only ~1 block worth of time on ethereum (assuming 12 seconds / block). Worth breaking out into a separate "ETH only" config? @mateusz-sekara